main.py

```diff
@@ -1,32 +1,17 @@
-
-#TODO Add red potato tracking to ignore
-#TODO Why does it exit after a few instead of sleeping?
-
 import pyautogui
+import cv2
 import time
 
 LOCATECOUNT = 0
 STARTFIND = 0 
 GAME_STARTED = False
 POTATO_COUNT = 0
 
-def clickWhack():
-    whackButton = pyautogui.locateOnScreen('images/960x540/whackAPotato.png')
-    if whackButton != None:
-        print('Move to Whack-A-Potato')
-        pyautogui.moveTo(whackButton)        
-        pyautogui.click()
-        print(' Clicked Whack-A-Potato')
-        time.sleep(2)
-
 def clickStart():
     global STARTFIND
     global GAME_STARTED
 
-    startButton = pyautogui.locateOnScreen('images/960x540/start.png',confidence=0.9)
-
-    if STARTFIND > 3:
-        raise ValueError("Can't see start button")
+    startButton = pyautogui.locateOnScreen('images/960x540/start.png',confidence=0.5)
```

Comment:
With the lowered confidence, my game keeps seeing the top red bar as the start button and hanging. 
```suggestion
    startButton = pyautogui.locateOnScreen('images/960x540/start.png',confidence=0.8)
```